docs/getting-started/installation.md

```diff
@@ -0,0 +1,119 @@
+Installation
+============
+
+Local Installation
```

Comment:
I would switch Local installation with Docker. 

We can control the docker container better than a local environment. Reduce issues by starting people due to wrong configured environments.